pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1091,18 +1061,36 @@ func (n *NetworkPolicyController) syncAddressGroup(key string) error {
 		utilsets.MergeString(addrGroupNodeNames, internalNP.SpanMeta.NodeNames)
 	}
 	memberSet := n.getAddressGroupMemberSet(addressGroup)
+	if addressGroup.Selector.NodeSelector != nil {
+		ms, nodes := n.addNodeSelectorMemberSet(addressGroup.Selector.NodeSelector)
+		memberSet = memberSet.Union(ms)
```

Comment:
I think an AddressGroup shouldn't contain PodIP and NodeIP at the same time to avoid complexity.

Reply:
Haven’t come up with a more appropriate solution yet if we don’t add Node IPs to `addressGroup`, or can we add a new field `NodeIPs` in `NetworkPolicyPeer` like `FQDNs` or `ToServices`, or add the NodeIPs to `IPBlocks`?